File: Algorithms/Sorting/SelectionSort/SelectionSort.py

def selectionSort(arr): 
    for i in range(0, len(arr)): 
        minIndex = i
        for j in range(i + 1,len(arr)):
            if(arr[j] < arr[minIndex]):
                minIndex = j
        if(minIndex != i):
            arr[minIndex], arr[i] = arr[i], arr[minIndex]
    return arr

File: Algorithms/Sorting/SelectionSort/SelectionSort.js

function selectionSort(arr){
    let min, temp;
    const length = arr.length;
    for(let i = 0; i < length - 1; i++){
        min = i;
        for(let j = i + 1; j < length; j++){
            if(arr[j] < arr[min])
                min = j;
        }

        if(min != i){
            temp = arr[min];
            arr[min] = arr[i];
            arr[i] = temp;
        }
    }
    return arr
}

module.exports = selectionSort;
